src/components/ModalInfo/ModalInfo.scss

```diff
@@ -0,0 +1,21 @@
+.ModalInfo.standalone {
+  margin-bottom: 0.8rem;
+}
+
+.ModalInfo {
+  display: flex;
+  margin-bottom: 1.6rem;
+
+  &-icon {
+    padding-right: 0.6rem;
+    padding-top: 0.2rem;
+
+    img {
+      display: block;
+    }
+  }
+
+  &-text {
+    font-size: 1.3rem;
+  }
+}
```

Comment:
On branch `release-13` there is a component named `AlertInfo` which is coming form the SLTP modal PR. I think `ModalInfo` component can be replaced by the `AlertInfo` component if that's okay.

Reply:
Hm, there really is a good component with a same functionality. By the way this task is going to be merge directly to master and goes off-release so I don't know a good way to take this component from release-13 don't you?
May we merge this PR as is and i will replace ModalInfo separately after release-13 merging?